utils/searchConfig.js

```diff
@@ -302,6 +302,11 @@ const config = {
         esFieldName: 'eventType.title.keyword',
         inputType: 'checkbox',
       },
+      {
```

Comment:
I thought when I wrote the ticket it will work, apologies for the confusion, please remove the past object, Elastic search is not setup with this filter from craft